MidasFrontend/ArduinoHelper.cxx

```diff
@@ -73,6 +92,10 @@ static int32_t mm_to_cts(float val_mm) {
     return round(val_mm / mm_cts_ratio);
 }
 
+static float cts_to_mm(int32_t val_cts) {
```

Comment:
should we make this `uint32_t`? I do remember from a previous PR you mentioned position is generalized so it may be positive or negative, but encoder counts are always positive right? Or does the system interpret negative counts as going in the opposite direction?

Reply:
Encoder counts can be negative, otherwise things can break with weird wrap-around conditions